com.unity.render-pipelines.high-definition/CHANGELOG.md

```diff
@@ -332,6 +332,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Changed light reset to preserve type.
 - Ignore hybrid duplicated reflection probes during light baking.
 - Updated the recursive rendering documentation (case 1338639).
+- Updated the Physically Based Sky documentation for baked lights.
```

Comment:
Capitalise "Lights".

Updated the Physically Based Sky documentation for baked Lights.